recipes/unicorn-binance-websocket-api/meta.yaml

```diff
@@ -45,7 +34,6 @@ requirements:
 
 test:
   imports:
-    - tools
```

Comment:
This is just removing it from the tests but the issue will still persist. In the end, you'll need to update this upstream.